1-org/modules/centralized-logging/variables.tf

```diff
@@ -25,15 +25,27 @@ variable "resources" {
 }
 
 variable "resource_type" {
-  description = "Resource type of the resource that will export logs to destination. Must be: project, organization, or folder."
+  description = "Resource type of the resource that will export logs to destination. Must be: project, organization, billing_account or folder."
   type        = string
 
   validation {
-    condition     = contains(["project", "folder", "organization"], var.resource_type)
-    error_message = "The resource_type value must be: project, organization, or folder."
+    condition     = contains(["project", "folder", "organization", "billing_account"], var.resource_type)
+    error_message = "The resource_type value must be: project, organization, billing_account or folder."
   }
 }
 
+variable "billing_account" {
```

Comment:
You need to pass an input to this variable in the module caller.

In order to do that, you should add a `billing_account = local.billing_account` input in `logs_export` module located in `/1-org/envs/shared/log_sinks.tf`.
Also, since `billing_account` will always be defined in 0-bootstrap, I don't think we should have a `default = ""` here. I would recommend no default. (If we had no default here, we would notice this missing input before due to an error)

Reply:
Done.